Solutions/Marain.Tenancy.Client/Marain/Tenancy/Client/ITenancyService.cs

```diff
@@ -116,7 +116,7 @@ public partial interface ITenancyService : System.IDisposable
         /// The name for the new tenant
         /// </param>
         /// <param name='wellKnownChildTenantGuid'>
-        /// The well known Guid for the new tenant. If provided, this will be used to create the child tenant Id.
+        /// The name for the new tenant
```

Comment:
Eh? Isn't that the description for `tenantName`? I think the old text was close to being correct, wasn't it? (Albeit slightly awkwardly-worded. And arguably it should say something about what happens if it's missing.)

Reply:
Oh hold on, this has happened because you reran `autorest` hasn't it? So that implies that the source of this mistake is actually the yaml?

Reply:
Have fixed the YAML.